app/src/main/java/com/android/voyageur/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -33,31 +34,40 @@ fun BottomNavigationMenu(
     onTabSelect: (TopLevelDestination) -> Unit,
     tabList: List<TopLevelDestination>,
     selectedItem: String?,
-    userViewModel: UserViewModel
+    userViewModel: UserViewModel,
+    tripsViewModel: TripsViewModel
 ) {
-  val notifications by userViewModel._notificationCount.collectAsState()
-  LaunchedEffect(notifications) { userViewModel.getFriendRequests {} }
+  val userNotifications = userViewModel.notificationCount?.collectAsState(initial = 0)?.value ?: 0
+  val tripInvites = tripsViewModel.tripNotificationCount?.collectAsState(initial = 0)?.value ?: 0
+  val totalNotifications = userNotifications + tripInvites
+
+  LaunchedEffect(userNotifications, tripInvites) {
+    userViewModel.getFriendRequests {}
+    tripsViewModel.getNotificationsCount {}
+  }
   LaunchedEffect(Unit) {
     if (Firebase.auth.uid != null) {
       userViewModel.getNotificationsCount {}
+      tripsViewModel.getNotificationsCount {}
```

Comment:
Nit: Couldn't we call `tripsViewModel.fetchTripInvites()` here as well? instead of a LaunchedEffect in the Profile Screen?